docs/products/valkey/howto/connect-valkey-cli.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Connect with valkey-cli
+---
+
+Learn how to establish a connection to an Aiven for Valkey™ service using the `valkey-cli`.
+
+
+## Variables
+
+Replace the following placeholders in the code sample with actual values
+from your service overview page:
+
+| Variable    | Description                                              |
+| ----------- | -------------------------------------------------------- |
+| `SERVICE_URI` | URI for the Aiven for Valkey service connection  |
+
+## Prerequisites
+
+For this example, you will need:
+
+1.  The `valkey-cli` client installed. You can install this as part of
+    [Valkey server installation](https://valkey.io/topics/installation/).
+
```

Comment:
```suggestion
Ensure that the valkey-cli client is installed. You can install it as part of the [Valkey server installation](https://valkey.io/topics/installation/).
```

According to Google's style guide, avoid lists with a single item.

Reply:
That was making me twitch, but it's how the page I copied was doing it. Glad to fix, especially as I think the replacement is much easier to read.